code/modules/mob/living/silicon/silicon.dm

```diff
@@ -57,6 +57,9 @@
 	return
 
 /mob/living/silicon/drop_item()
```

Comment:
robots generally can't drop items, what's this abour

Reply:
it "drops" active module from panel, storing it